src/NServiceBus.RavenDB/NServiceBus.RavenDB.csproj

```diff
@@ -15,7 +15,7 @@
     <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.1897, 9.0.0)" />
     <PackageReference Include="Obsolete.Fody" Version="5.2.1" PrivateAssets="All" />
     <PackageReference Include="Particular.Packaging" Version="1.3.0" PrivateAssets="All" />
-    <PackageReference Include="RavenDB.Client" Version="[4.2.0, 5.0.0)" />
+    <PackageReference Include="RavenDB.Client" Version="[5.2.1, 6.0.0)" />
```

Comment:
Shouldn't this be 5.2.0?

Reply:
Ideally yes. The problem is though that 5.2.0 has a bug that prevents empty strings from being serialized on the .net framework. Given that we multi target also for the .net framework I think we should do our customers a service to prevent them from running into this problem (found it out because the persistence test didn't have an explicit RavenDB Client reference)